src/MSBuild/Resources/Strings.resx

```diff
@@ -756,9 +755,18 @@
     </comment>
   </data>
   <data name="HelpMessage_35_IsolateProjectsSwitch" UESanitized="false" Visibility="Public">
-    <value>  -isolateProjects[:True|False]
+    <value>  -isolateProjects[:True|MessageUponIsolationViolation|False]
                      Causes MSBuild to build each project in isolation.
 
+                     When set to MessageUponIsolationViolation (or its short
+                     form Message), only the results from top-level targets
+                     are serialized if the -orc switch is supplied to mitigate
```

Comment:
What is orc? Also, can you split this sentence into 2-3 sentences?

Reply:
`-orc` is short for `-outputResultsCache`. I'll use the full name for clarity.